generators/ci-cd/templates/azure-pipelines.yml.ejs

```diff
@@ -36,6 +37,16 @@ jobs:
     inputs:
       versionSpec: $(NODE_VERSION)
     displayName: 'TOOLS: install Node.js'
+<%_ } _%>
+<%_ if (testFrameworks.includes('cypress') || testFrameworks.includes('protractor')) {
+  const commented = applicationType === 'gateway' ? '# ' : ''; _%>
+  <%= commented %>- task: Bash@3
```

Comment:
To keep consistency, we could do the same than in our main azure-pipelines, see https://github.com/jhipster/generator-jhipster/blob/main/azure-pipelines.yml#L112-L117

```yaml
      - script: |
          wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
          sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
          sudo apt update
          sudo apt install google-chrome-stable
        displayName: 'TOOLS: install google-chrome-stable'
```

What's the best in your opinion ?
- the code you propose here
- or the one in our CI ?

Reply:
I would prefer the code I proposed, because Azure uses "throw away VMs" and it is not necessary to install the keys, update apt and so on...
But to keep consistency I can change it.

What do you think about the `--no-sandbox` flag? Should I also include these changes?

Reply:
I remember having some issue with no-sandbox... is it necessary ? If not, we can keep like this

Reply:
We couldn't get it to run without the flag.
I will investigate further and do some tests, maybe there is another way. Let's keep it like this for now.

Reply:
Then, if it's needed, so we should add no-sandbox so